tests/manual-test-cases/Group5-Functional-Tests/5-5-Heterogeneous-ESXi.robot

```diff
@@ -37,7 +37,9 @@ Test
     Append To List  ${list}  ${esx3}
 
     # Finish vCenter deploy
-    ${output}=  Wait For Process  ${pid-vc}
+    ${output}=  Wait For Process  ${pid-vc}  timeout=40 minutes  on_timeout=terminate
```

Comment:
we can eventually, I am not inclined to pollute this PR with changes throughout every single test though. we can file a ticket to go through and do that later.

https://github.com/vmware/vic/issues/5908